webui/react/src/hooks/useFeature.ts

```diff
@@ -37,6 +38,7 @@ export const FEATURES: Record<ValidFeature, FeatureDescription> = {
     defaultValue: false,
```

Comment:
also want `defaultValue` to be `true`, right?

Reply:
Actually we don't want this always on, because we still want `featureSwitches` from `/api/v1/master` to be able to return and control whether genai is supported or not. If the deployment does not support genai, then `featureSwitches` will not have `genai` in the list and the UI will not show the corresponding link either. Having it on makes it always appear no matter what for me.